src/Network/Network/ChangeLog.md

```diff
@@ -19,11 +19,13 @@
 --->
 
 ## Upcoming Release
-
-## Version 4.12.0
-* Support for Sku, ScaleUnits parameters of BastionHost resource.
+* Added optional parameter `-IsSecuritySite` to the following cmdlet:
+    - `New-AzVpnSite`
+* Support for Sku, ScaleUnits parameters of BastionHost resource. [#15885]
     - `New-AzBastion`
     - `Set-AzBastion`
+
+## Version 4.12.0
 * Onboard Azure Resource Manager to Private Link Common Cmdlets
 * Updated cmdlets to add properties to enable/disable BgpRouteTranslationForNat for VpnGateway.
     - `New-AzVpnGateway`
```

Comment:
It seems the changelog is still a bit off. "Support for Sku... Set-AzBastion" part should be in version 4.12.0 instead of upcoming release.
Please check again, make sure "upcoming release" contains unpublished changes and published versions are not touched.